test/CDB_JenksBinsTest.sql

```diff
@@ -1,11 +1,23 @@
 WITH data AS (
-    SELECT array_agg(x::numeric) s FROM generate_series(1,300) x
-        WHERE x % 5 != 0 AND x % 7 != 0
-    ) 
-SELECT unnest(CDB_JenksBins(s, 7)) FROM data;
+    SELECT Array[0.99, 1.0, 1.01,
+                 4.99, 5.01,
+                 10.01, 10.01,
+                 15.01, 14.99,
+                 20.1, 19.9]::numeric[] AS s
+)
+-- expectation is: 1, 5, 10, 15, 20
+-- TODO: fix cdb_jenksbins to match ^^
+SELECT round(unnest(CDB_JenksBins(s, 5))) FROM data;
```

Comment:
Why are we changing the JenkBins tests? Are we using the `QuantileBins` function internally in the Jenks ones?

Reply:
Jenks is dependent on Quantiles for setting the initial bin edges, and then it tries to do it's jenksy calculations from there.

It turns out that Jenks is 'incorrect' and heavily dependent on the initial bin edges. Getting QuantileBins function correct is the first start of getting Jenks fixed.

Reply:
I see. So everything is good then :)